.travis.yml

```diff
@@ -7,6 +7,13 @@ dist: trusty
 
 matrix:
   include:
+  - go: "1.14.x"
+    env:
+    - TESTS=true
+    - COVERAGE=true
+    arch: ppc64le
+    addons:
+        apt: jq
```

Comment:
why is this needed?

Reply:
why is this whole section needed? We do not run tests per architecture, it's too expensive without significant benefits.

Reply:
It is for executing unit test on ppc64le but since it is expensive, I have removed it as requested.